extensions/emmet/src/extension.ts

```diff
@@ -137,9 +128,35 @@ export function activate(context: vscode.ExtensionContext) {
 	resolveUpdateExtensionsPath();
 
 	context.subscriptions.push(vscode.workspace.onDidChangeConfiguration(() => {
+		registerCompletionProviders(context, false);
 		resolveUpdateExtensionsPath();
 	}));
 }
 
+/**
+ * Holds any registered completion providers by their language strings
+ */
+const registeredCompletionProviders: string[] = [];
+
+function registerCompletionProviders(context: vscode.ExtensionContext, isFirstStart: boolean) {
+	if (isFirstStart) {
+		let completionProvider = new DefaultCompletionItemProvider();
```

Comment:
This `completionProvider` assignment needs to be pulled out of the `if` block so that the `else` block can use it as well